longview.go

```diff
@@ -2,16 +2,53 @@ package linodego
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
+	"time"
 
 	"github.com/go-resty/resty/v2"
+	"github.com/linode/linodego/internal/parseabletime"
 )
 
 // LongviewClient represents a LongviewClient object
 type LongviewClient struct {
```

Comment:
One last small thing; in order to get the Created and Updated fields to unmarshal correctly, you'll need to implement an `UnmarshalJSON` method like we do here: https://github.com/linode/linodego/blob/main/stackscripts.go#L68-L88